zerver/lib/outgoing_webhook.py

```diff
@@ -76,15 +119,17 @@ def do_rest_call(rest_operation, event, timeout=None):
     request_kwargs['timeout'] = timeout
 
     try:
-        response = requests.request(http_method, final_url, data=json.dumps(event), **request_kwargs)
+        response = requests.request(http_method, final_url, data=json.dumps(request_data), **request_kwargs)
```

Comment:
Just double checking: after changing the payload from `event` to `request_data`, will the Flask bot server still handle the request correctly?